usaspending_api/etl/data_diff_tool.py

```diff
@@ -0,0 +1,42 @@
+from gresearch.spark.diff import *
+
+def data_diff_function(df1, df2):
+    #assumes dataframes as a delta table
+    table1 = spark.read.load(df1)
+    table2 = spark.read.load(df2)
+    return (table1.diff(table2, table1.columns).show)
```

Comment:
I'm guessing that this method makes use of the gresearch library? What does this method provide that the `diff` method below does not? Is it necessary?

Reply:
Concur @bkschmidt I think we remove this function. Folks can use it ad-hoc in notebooks if needed, but I'd rather not have our code having a dependency on this library being pip installed. As it stands right now, it would fail since there is no update to requirements.txt that would bring the dependent library into a development env.